PortMaster/control.txt

```diff
@@ -158,3 +158,10 @@ if [ -f "$controlfolder/cwtbe_flag" ]; then
 else
   GPTOKEYB="$ESUDO $controlfolder/gptokeyb $ESUDOKILL"
 fi
+
+# Set current virtual screen
```

Comment:
muOS uses the file `PortMaster/muos/control.txt`, not `PortMaster/control.txt`. :)

Reply:
Geez just close this